pytest_opentmi/OpenTmiReport.py

```diff
@@ -297,6 +297,9 @@ def pytest_runtest_logreport(self, report):
                 self._append_inconclusive(report)
             elif report.skipped:
                 self._append_skipped(report)
+        elif report.when == 'teardown':
+            if report.failed:
```

Comment:
this probably generate second report instead single one, first one is appended during test 'call' phase. Not accepted I think